### PR TITLE
Docs:add-workflow-case-全能助手

### PR DESCRIPTION
**Pull request by astorm-z**
新增高级编排案例：全能助手

### CODE REVIEW

**Comment by c121914yu** [comment]
# 🤖 Generated by deploy action
[👀 Visit Preview](https://fastgpt-staging.vercel.app)

File: docSite/content/docs/workflow/examples/versatile_assistant.md [hunk 1]
```diff
@@ -0,0 +1,1535 @@
+---
+title: '全能助手'
+description: '赋予联网功能，将用户问题进行分类，细分后对接对应api获取信息，经GPT整理后返回'
```
Comment (yangchuansheng) [inline docSite/content/docs/workflow/examples/versatile_assistant.md:3]: Put a space between Chinese and English

File: docSite/content/docs/workflow/examples/versatile_assistant.md [hunk 2]
```diff
@@ -0,0 +1,1535 @@
...
+## 简要介绍一下“全能助手”的思路
+
+思路说来也简单，以下分别用文字和图片两种方式介绍下
+
+- **文字描述：**
+
+1. 对于用户输入的问题，通过【问题分类】模块进行区分，分出【询问天气】、【微博热搜】、【其他问题】等
+2. 对于【询问天气】的情况，调用第三方 api 查询天气（后文会介绍），将查询到的 json 结果丢给【AI 对话】模块，让它根据用户问题来给出回答
+3. 对于【微博热搜】的情况，同理，也是调的第三方 api
+4. 对于【其他问题】的情况，直接走【AI 对话】模块就好了，跟普通的 gpt 聊天一样
```
Comment (yangchuansheng) [inline docSite/content/docs/workflow/examples/versatile_assistant.md:25]: change all "api" to "API"
change all "gpt" to "GPT"

File: docSite/content/docs/workflow/examples/versatile_assistant.md [hunk 3]
```diff
@@ -0,0 +1,1535 @@
...
+```
+
+3. 由于 fastgpt 的 http 模块，对于返回的 json 是以对象形式接收，而我们期望得到的是上述 json 中的“data”字段，而“data”又是数组格式，无法直接丢给【AI 对话】模块（我丢过，非字符串格式报错了，不知道后面会不会更新），所以需要对其做一层中转，将“data”字段转成字符串格式。思路如此，中转方式多样，这里介绍我自己的做法：用 python 起一个服务，来负责对 api 的中转，代码如下（包含了天气接口和微博热搜接口）：
```
Comment (yangchuansheng) [inline docSite/content/docs/workflow/examples/versatile_assistant.md:53]: change all "fastgpt" to "FastGPT"

**Review by yangchuansheng — APPROVED** [review]

**Pull request merged**